lib/user-interface/react/src/components/model-management/create-model/BaseModelConfig.tsx

```diff
@@ -124,11 +125,11 @@ export function BaseModelConfig (props: FormProps<IModelRequest> & BaseModelConf
                             props.setFields({'streaming': detail.checked})
                         }
                         onBlur={() => props.touchFields(['streaming'])}
-                        disabled={props.item.modelType === ModelType.embedding}
+                        disabled={props.item.modelType === ModelType.embedding || props.item.modelType === ModelType.imagegen}
```

Comment:
I see this a few times and it would be easier to understand and cleaner to pull this out into a reusable variable.